src/database/simple.jl

```diff
@@ -394,6 +394,34 @@ const renames = [
     @add_rename iteratoreltype IteratorEltype   v"0.7.0-DEV.3309"
     @add_rename nb_available   bytesavailable   v"0.7.0-DEV.3477"
     @add_rename broadcast_indices broadcast_axes  v"0.7.0-DEV.4936"
+    @add_rename Base.Test      Test             v"0.7.0-alpha" 
+    @add_rename LinAlg         LinearAlgebra    v"0.7.0-alpha"
+    @add_rename bits           bitstring        v"0.7.0-alpha"
+    @add_rename search         findfirst        v"0.7.0-alpha"
+    @add_rename rsearch        findlast         v"0.7.0-alpha"
+    @add_rename Random.dSFMT   Random.DSFMT     v"0.7.0-alpha"
+    @add_rename gc             GC.gc            v"0.7.0-alpha"
+    @add_rename gc_enable      GC.enable        v"0.7.0-alpha"
+    @add_rename Base.@gc_preserve GC.@preserve  v"0.7.0-alpha"
+    @add_rename LinSpace      LinRange          v"0.7.0-alpha"
+    @add_rename endof         lastindex         v"0.7.0-alpha"
+    @add_rename DateTime()    DateTime(1)       v"0.7.0-alpha"
+    @add_rename Date()        Date(1)           v"0.7.0-alpha"
+    @add_rename Time()        Time(1)           v"0.7.0-alpha"
+    @add_rename isupper       isuppercase       v"0.7.0-alpha"
+    @add_rename islower       islowercase       v"0.7.0-alpha"
+    @add_rename ucfirst       uppercasefirst    v"0.7.0-alpha"
+    @add_rename lcfirst       lowercasefirst    v"0.7.0-alpha"
+    @add_rename Base.IntSet   Base.BitSet       v"0.7.0-alpha"
+    @add_rename linearindices LinearIndices     v"0.7.0-alpha"
+    @add_rename LinAlg.fillslots! LinearAlgebra.fillstored! v"0.7.0-alpha"
+    @add_rename cfunction      @cfunction       v"0.7.0-alpha"
+    @add_rename GMP.gmp_version GMP.version     v"0.7.0-alpha"
```

Comment:
This needs support for renames in other modules, which this doesn't provide. Remove these for now.